_timezones.toml

```diff
@@ -54,3 +54,20 @@ start = "1970-03-29T02:00:00.00"
 from = "+0100"
 to = "+0200"
 rule = { frequency = "yearly", by_month = [3], by_day = [ { day = "sunday", num = -1 } ] }
+
+[[timezones]]
+name = "Europe/Paris"
```

Comment:
You don't need to add this, you can just use `Europe/Amsterdam` instead, it amounts to the same timezones.